src/main/java/com/iexec/core/configuration/ResultRepositoryConfiguration.java

```diff
@@ -36,8 +38,11 @@ public String getResultRepositoryURL() {
         return protocol + "://" + host + ":" + port;
     }
 
-    @Bean
-    public ResultProxyClient resultProxyClient() {
-        return ResultProxyClientBuilder.getInstance(Logger.Level.NONE, getResultRepositoryURL());
+    public ResultProxyClient createProxyClientFromURL(final String url) {
```

Comment:
I think we prefer to be as precise as possible here.
A proxy is far to generic, many things can be proxied in computer science
```suggestion
    public ResultProxyClient createResultProxyClientFromURL(final String url) {
```